web/html/src/manager/minion/ansible/accordion-path-content.tsx

```diff
@@ -0,0 +1,182 @@
+            this.setState({
+              content: this.props.path.type === "playbook" ?
+                this.digestPlaybookPathContent(data)
+                :
+                this.digestInventoryPathContent(data)
+            });
+          }
+          else {
+            this.setState({ errors: blob.messages });
+          }
+          this.setState({ open: true, loading: false });
+        });
+      }
+      else {
+        this.setState({ open: true });
+      }
+    }
+    else {
+      this.setState({ open: false, errors: [] });
+    }
+  }
+
+  digestPlaybookPathContent(blob: any) {
+    const content: Map<string, {}> = blob[this.props.path.path];
+    const playbookNameList: string[] = Object.keys(content);
+
+    const playbookDetailsList: PlaybookDetails[] = [];
+    playbookNameList.forEach(playbookName => {
+        const playbookObj = content[playbookName];
+        const playbookObjKeys = Object.keys(playbookObj);
+        const newPlaybookObject: PlaybookDetails = {
+          name: playbookName,
+          fullPath: playbookObjKeys.includes("fullpath") ? playbookObj["fullpath"] : "-",
+          path: this.props.path,
+          customInventory: playbookObjKeys.includes("custom_inventory") ? playbookObj["custom_inventory"] : "-"
+        }
+        playbookDetailsList.push(newPlaybookObject);
+      }
+    );
+
+    return playbookDetailsList;
+  }
+
+  renderPlaybookPathContent() {
+    if (this.state.content === null || this.state.content.length === 0) {
+      return <div>{t("No Playbook found.")}</div>
+    }
+
+    const content: PlaybookDetails[] = this.state.content;
+    return content.map((p, i) =>
+      <div key={p.toString() + "_" + i.toString()}>
+        { i === 0 ? <br/> : null }
+        <dl className="row">
+          <dt className="col-xs-2">{t('Playbook File Name')}:</dt>
+          <dd className="col-xs-8">{p.name}</dd>
+        </dl>
+        <dl className="row">
+          <dt className="col-xs-2">{t('Full Path')}:</dt>
+          <dd className="col-xs-8">{p.fullPath}</dd>
+        </dl>
+        <dl className="row">
+          <dt className="col-xs-2">{t('Custom Inventory')}:</dt>
+          <dd className="col-xs-8">{p.customInventory}</dd>
+        </dl>
+        { i < content.length -1 ? <hr/> : null }
+      </div>
+    );
+  }
+
+  digestInventoryPathContent(blob: any) {
+    return blob
+  }
+
+  renderInventoryPathContent() {
+    if (this.state.content === null || this.state.content.length === 0) {
+      return <div>{t("Inventory file not found or empty.")}</div>
+    }
+
+    return <AceEditor
+              className="form-control"
+              id="content-state"
+              minLines={20}
+              maxLines={40}
+              readOnly={true}
+              mode="yaml"
+              content={this.state.content}
+            ></AceEditor>;
+  }
+
+  render() {
+    const header =
+      <div className="panel-heading pointer" onClick={() => this.onToggle()}>
```

Comment:
Would be nice if this was a `<button />` element instead, but this is a small nitpick that can be revisited later.